### PR TITLE
New client form

### PR DESCRIPTION
**Pull request by yukim17**
Simulator Screen Shot - iPhone 14 - 2023-01-12 at 16 29 30

### CODE REVIEW

File: SlimApp/UI/NewClient/NewClientView.swift [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                Section {
+                    TextField("Name", text: $viewModel.name)
+                    TextField("Surname", text: $viewModel.surname)
+                    TextField("E-mail", text: $viewModel.email)
+                        .keyboardType(.emailAddress)
+                    TextField("Phone number", text: $viewModel.phoneNumber)
+                        .keyboardType(.phonePad)
+                }
+                Section {
+                    TextField("Height", text: $viewModel.height)
```
Comment (palant-dev) [inline SlimApp/UI/NewClient/NewClientView.swift:27]: Should we use something like a Picker here?

File: SlimApp/UI/NewClient/NewClientView.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                }
+                Button {
+                    #warning("Implement saving client information")
+                } label: {
+                    Text("Save")
+                        .frame(width: UIScreen.main.bounds.width)
+                }
+                .buttonStyle(.borderless)
+            }
+            .navigationTitle("New client")
```
Comment (palant-dev) [inline SlimApp/UI/NewClient/NewClientView.swift:48]: Maybe here we could "force" the title as a '.inline' to recover some space in the view.

File: SlimApp/UI/NewClient/NewClientViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,20 @@
+//
+//  NewClientViewModel.swift
+//  SlimApp
+//
+//  Created by Ekaterina Grishina on 11/01/23.
+//
+
+import Foundation
+
+class NewClientViewModel: ObservableObject {
```
Comment (palant-dev) [inline SlimApp/UI/NewClient/NewClientViewModel.swift]: Maybe here would be better to have a final class in order to restrict future undesired inheritances.

File: SlimApp/UI/NewClient/NewClientView.swift (hunk 1 above)
Comment (yukim17) [reply SlimApp/UI/NewClient/NewClientView.swift:27]: For now let's keep it with TextField. I'll come up later with solution how to make picker look nice here

File: SlimApp/UI/NewClient/NewClientView.swift (hunk 2 above)
Comment (yukim17) [reply SlimApp/UI/NewClient/NewClientView.swift:48]: Large title looks better here, imho. We can discuss

File: SlimApp/UI/NewClient/NewClientViewModel.swift (hunk 3 above)
Comment (yukim17) [reply SlimApp/UI/NewClient/NewClientViewModel.swift]: Done

**Pull request merged**